src/templates/header.html

```diff
@@ -1,72 +1,113 @@
 <a class="usa-skipnav" href="#main-content">Skip to main content</a>
-<div class="usa-banner usgs-banner">
-    <header class="usa-banner-header">
-        <div class="usa-banner-inner">
-            <div class="grid-col-auto">
-                <img class="usa-banner-header-flag" src="{{ uswds.path }}img/favicons/favicon-57.png" alt="U.S. flag">
+
+<section class="usa-banner">
+    <div class="usa-accordion">
+        <header class="usa-banner__header">
+            <div class="usa-banner__inner">
+                <div class="grid-col-auto">
+                    <img class="usa-banner__header-flag" src="{{ uswds.path }}/img/us_flag_small.png" alt="U.S. flag">
```

Comment:
I think you need to leave out the '/' right before img. If uswds.path is used it should add with a '/'.

Reply:
Okay. changed that. I also changed a similar thing on lines 23 and 35.